plutus-core/cost-model/budgeting-bench/Common.hs

```diff
@@ -84,7 +84,9 @@ benchWith
 -- the result, so e.g. 'evaluateCek' won't work properly because it returns a pair whose components
 -- won't be evaluated by 'whnf'. We can't use 'nf' because it does too much work: for instance if it
 -- gets back a 'Data' value it'll traverse all of it.
-benchWith params name term = bench name $ whnf (evaluateCekNoEmit params) term
+benchWith params name term =
+  bench name $
+    whnf (unsafeSplitStructuralOperational . evaluateCekNoEmit params) term
```

Comment:
This makes a benchmark fail if evaluation fails (wasn't the case before)

Reply:
Maybe you could add a comment to remind us why `unsafeSplitStructuralOperational` is there, since it's not immediately obvious what it's doing.